web/public/index.html

```diff
@@ -45,7 +45,7 @@
           env: getTemplateValue('{{ .Env }}', ''),
           demoEnabled: getTemplateValue('{{ .DemoEnabled }}', '["pokeshop", "otel"]'),
           demoEndpoints: getTemplateValue('{{ .DemoEndpoints }}', '{"PokeshopHttp": "http://demo-pokemon-api.demo.svc.cluster.local", "PokeshopGrpc": "demo-pokemon-api.demo.svc.cluster.local:8082", "OtelFrontend": "http://otel-frontend:8084" }'),
-          experimentalFeatures: getTemplateValue('{{ .ExperimentalFeatures }}', '[]'),
+          experimentalFeatures: getTemplateValue('{{ .ExperimentalFeatures }}', '["transactions1"]'),
```

Comment:
Do we want to keep it enabled by default now?

Reply:
I think so!

Reply:
This was a mistake. Fixed!